### ISSUE TITLE
 restore failed at Oracle Linux 8.5

### ISSUE THREAD

**Issue by hiroaka73** [rear/rear#2856]
#### Relax-and-Recover (ReaR) Issue Template

Fill in the following items before submitting a new issue
(quick response is not guaranteed with free support):

* ReaR version ("/usr/sbin/rear -V"):  2.6 / 2020-06-17

* OS version ("cat /etc/os-release" or "lsb_release -a" or "cat /etc/rear/os.conf"):  Orcle Linux 8.5

* ReaR configuration files ("cat /etc/rear/site.conf" and/or "cat /etc/rear/local.conf"):

* Hardware vendor/product (PC or PowerNV BareMetal or ARM) or VM (KVM guest or PowerVM LPAR):

* System architecture (x86 compatible or PPC64/PPC64LE or what exact ARM device):

* Firmware (BIOS or UEFI or Open Firmware) and bootloader (GRUB or ELILO or Petitboot):

* Storage (local disk or SSD) and/or SAN (FC or iSCSI or FCoE) and/or multipath (DM or NVMe):

* Storage layout ("lsblk -ipo NAME,KNAME,PKNAME,TRAN,TYPE,FSTYPE,LABEL,SIZE,MOUNTPOINT"):

* Description of the issue (ideally so that others can reproduce it):

* Workaround, if any:
[20220906_140817.log](https://github.com/rear/rear/files/9493461/20220906_140817.log)

* Attachments, as applicable ("rear -D mkrescue/mkbackup/recover" debug log files):

To paste verbatim text like command output or file content,
include it between a leading and a closing line of three backticks like
````
```
verbatim content
```
````

**Comment by jsmeix** [comment]
https://github.com/rear/rear/files/9493461/20220906_140817.log
contains at its beginning and at its end (excerpts):
```
2022-09-06 21:26:31.936371680 Relax-and-Recover 2.6 / 2020-06-17
.
.
.
2022-09-06 21:29:07.064958617 Finished running recover workflow
2022-09-06 21:29:07.070062753 Exiting rear recover (PID 1755) and its descendant processes ...
2022-09-06 21:29:10.086844505 rear,1755 /bin/rear recover
  `-rear,24425 /bin/rear recover
      `-pstree,24426 -Aplau 1755
/usr/share/rear/lib/_input-output-functions.sh: line 151: kill: (24429) - No such process
2022-09-06 21:29:10.106399443 Running exit tasks
2022-09-06 21:29:10.143181873 Finished in 159 seconds
2022-09-06 21:29:10.145320740 Removing build area /tmp/rear.trSlqgb0gtB0EY1
rmdir: removing directory, '/tmp/rear.trSlqgb0gtB0EY1/outputfs'
rmdir: failed to remove '/tmp/rear.trSlqgb0gtB0EY1/outputfs': Directory not empty
2022-09-06 21:29:10.204966314 ERROR: 
====================
BUG in /usr/share/rear/lib/framework-functions.sh line 138:
'Directory /tmp/rear.trSlqgb0gtB0EY1/outputfs not empty, can not remove'
--------------------
Please report this issue at https://github.com/rear/rear/issues
and include the relevant parts from /var/log/rear/rear-localhost.log
preferably with full debug information via 'rear -D recover'
====================
===== Stack trace =====
Trace 0: /bin/rear:1 main
Trace 1: /usr/share/rear/lib/_input-output-functions.sh:321 DoExitTasks
Trace 2: /usr/share/rear/lib/framework-functions.sh:138 cleanup_build_area_and_end_program
Trace 3: /usr/share/rear/lib/_input-output-functions.sh:708 BugError
=== End stack trace ===
```

Please test if it works for you when you use
our current ReaR GitHub master code.

See the section
"Testing current ReaR upstream GitHub master code" in
https://en.opensuse.org/SDB:Disaster_Recovery
how you can try out our current ReaR GitHub master code
without conflicts with your already installed ReaR version.

In general I recommend to try out our latest GitHub master code
because the GitHub master code is the only place where we fix things
and if there are issues it helps when you use exactly the code
where we could fix things.

**Comment by hiroaka73** [comment]
Thanks for your comment.
I will try the information you provided.

**Comment by pcahyna** [comment]
Hello, I suspect this is a duplicate of https://github.com/rear/rear/issues/2676 that should be fixed by #2675
